internal/app/stream/multiplexer.go

```diff
@@ -0,0 +1,34 @@
+package multiplexer
```

Comment:
do you prefer this to live in the server package instead?

Reply:
No, I think what you have is good. `server` was intended for the xDS relay gRPC server implementation. Thoughts on `app/upstream/multiplexer.go` vs `/app/stream/...` though?

Reply:
yeah `upstream` sounds much better.